<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-LBN8TGD" ignored="0" name="Default test(failed)" tests="1" failures="1" timestamp="29 Mar 2019 12:00:49 GMT" time="0.033" errors="0">
  <testcase name="run" time="0.008" classname="testng.TestRetry1">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng.TestRetry1.run(TestRetry1.java:11)
at testng.MultRetry.main(MultRetry.java:15)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- run -->
</testsuite> <!-- Default test(failed) -->
